net/netfilter/nf_tables_api.c

```diff
@@ -954,6 +954,25 @@ static void nf_tables_table_disable(struct net *net, struct nft_table *table)
 #define __NFT_TABLE_F_UPDATE		(__NFT_TABLE_F_WAS_DORMANT | \
 					 __NFT_TABLE_F_WAS_AWAKEN)
 
+static bool nft_table_pending_update(const struct nft_ctx *ctx)
+{
+	struct nftables_pernet *nft_net = nft_pernet(ctx->net);
```

Comment:
The RHEL doesn't use this and the use of it later causes build errors

Reply:
Oversight... I'll fix it up.